### PR TITLE
https://issues.redhat.com/browse/ACM-11428 z-stream ACM 2.10.3

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
I'm assuming you are taking this one, @dockerymick, since you added yourself as a reviewer. Feel free to take a look at it. There are more on this one, and a couple might need a couple of read-throughs. Thanks!

**Review by rhaignere — APPROVED** [review]
LGTM, thank you!

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,26 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* Fixes an issue when a policy was recreated quickly after deletion, it sometimes led to the compliance status not being populated on the managed cluster. (link:https://issues.redhat.com/browse/ACM-10664[ACM-10664])
+
+* Fixes an issue that caused some unnecessary logs in the `governance-policy-framework` pod to be displayed with the default log verbosity settings. (link:https://issues.redhat.com/browse/ACM-10693[ACM-10693])
+
+* Fixes an issue that required a restart of the `governance-policy-framework` pod when the Gatekeeper operator was installed or uninstalled. The {product-title} integration with Gatekeeper is now enabled or disabled without restarting the pod. (link:https://issues.redhat.com/browse/ACM-10966[ACM-10966])
+
+* Fixes a bug where a `ConfigurationPolicy` with the `MustOnlyHave` compliance type did not factor root-level keys in the object on the cluster when comparing it to the policy definition. (link:https://issues.redhat.com/browse/ACM-10877[ACM-10877])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes a bug where a `ConfigurationPolicy` resource with the `MustOnlyHave` compliance type did not factor root-level keys in the object on the cluster when comparing it to the policy definition. (link:https://issues.redhat.com/browse/ACM-10877[ACM-10877])
```

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -7,6 +7,26 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* Fixes an issue in the Policy Generator where some placement overrides that exist outside of `policyDefaults` did not correctly override the default. (link:https://issues.redhat.com/browse/ACM-11075[ACM-11075])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes an issue in the Policy Generator where some placement overrides that exist outside of the `policyDefaults` parameter section did not correctly override the default. (link:https://issues.redhat.com/browse/ACM-11075[ACM-11075])
```

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -7,6 +7,26 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* Fixes an issue that prevented the {product-title-short} `gitopsCluster` controller from automatically generating the managed cluster secret for the Argo CD push model when an application addon service account named `application-manager` was provisioned in {ocp} 4.15 by some cloud providers. (link:https://issues.redhat.com/browse/ACM-11149[ACM-11149])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes an issue that prevented the {product-title-short} `gitopsCluster` controller from automatically generating the managed cluster secret for the Argo CD push model when an application add-on service account named `application-manager` was provisioned in {ocp} 4.15 by some cloud providers. (link:https://issues.redhat.com/browse/ACM-11149[ACM-11149])
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made some suggestions

**Review by rhaignere — APPROVED** [review]
Thanks for updating! LGTM

**Review by dockerymick — APPROVED** [review]
Thanks Chris!

**Pull request merged**